build.proj

```diff
@@ -34,8 +34,9 @@
 
     <Stage0Directory>$(RepoRoot)/.dotnet_stage0/$(Architecture)</Stage0Directory>
     <DotnetStage0>$(Stage0Directory)/dotnet$(ExeExtension)</DotnetStage0>
-
     <DotnetCliBuildDirectory>$(RepoRoot)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
+    <ArtifactsDirectory>$(RepoRoot)/artifacts</ArtifactsDirectory>
+    <PackagesDir>$(RepoRoot)/.nuget</PackagesDir>
```

Comment:
PackagesDir and PackagesDirectory need to have more distinct naming. The path referred to here is equivalent to NuGetPackagesDir. Can we use that one instead?